packages/sdk/src/lib/dataProtectorSharing/smartContract/preflightChecks.ts

```diff
@@ -384,11 +384,17 @@ export const onlyAccountWithMinimumBalance = ({
   accountDetails,
   minimumBalance,
 }: {
-  accountDetails: { balance: bigint };
+  accountDetails: { stackedBalance: bigint; balance: bigint };
   minimumBalance: ethers.BigNumberish;
 }) => {
   const requiredBalance = BigInt(minimumBalance);
-  if (accountDetails.balance < requiredBalance) {
+  if (accountDetails.stackedBalance < requiredBalance) {
+    // The user have xRLC but he didn't stack them
+    if (accountDetails.balance > BigInt(0)) {
+      throw new Error(
+        `No xRLC stacked. To interact with the iExec protocol, you need to stake some xRLC. Visit https://explorer.iex.ec/bellecour, log in with your Wallet that your are using in this dev tool, and stake xRLC in the iExec protocol.`
```

Comment:
Message to be improved?
We probably don't want to redirect the user to the explorer?

Reply:
```suggestion
        `No xRLC in your iExec account. You may have xRLC in your wallet but to interact with the iExec protocol, you need to deposit some xRLC into your iExec account.`
```